docs/UserGuide.md

```diff
@@ -450,9 +585,23 @@ Format: `exit`
 
 #### 4.3 Saving the data :
 
-All tCheck data (i.e. contact details, ingredient data, sales data) are saved in the hard disk automatically after any
- command that changes the data. There is no need to save manually by entering any command.
+All tCheck data (i.e. ingredient data, sales data, employees' data) will be saved in the hard disk as json
+ file automatically after each command executes. There is no need to save manually by entering any command.
```

Comment:
I think the older version is better and less technical.

Reply:
ok, thanks